qlty-cli/src/commands/coverage/publish.rs

```diff
@@ -303,6 +311,8 @@ impl Publish {
             eprintln_unless!(self.quiet, "    total-parts-count: {}", total_parts_count);
         }
 
+        eprintln_unless!(self.quiet, "    incomplete: {}", self.incomplete);
```

Comment:
Generally we only print settings provided by the user, to balance signal vs noise. In this case, that means I would probably omit this line unless the arg is provided.